wait.sh

```diff
@@ -79,4 +79,7 @@ function wait() {
 echo "Waiting for services to start ..."
 wait
 echo
+echo "Sync Elasticsearch"
```

Comment:
The wait script is not the right place for this.  Consider adding a separate makefile rule to re-run sync externally as required, and call that after the services are known to be running.

Further, this is probably the cause of the CI failure - you're trying to sync before the configuration is applied

Reply:
Good point! I did the changes, added a new target just for elastic. I left the host configuration on the config target, since this only needs to run once.